src/frontend/src/optimizer/rule/index_selection.rs

```diff
@@ -59,8 +66,9 @@ const INDEX_COST_MATRIX: [[usize; INDEX_MAX_LEN]; 4] = [
     [1, 1, 1, 1, 1],
     [10, 8, 5, 5, 5],
     [500, 50, 20, 10, 10],
-    [10000, 100, 50, 20, 20],
+    [10000, 100, 30, 20, 20],
```

Comment:
Why bother to change this? Or where it come from?

Reply:
For this case:
```
    set rw_batch_enable_lookup_join=true;
    set query_mode=local;
    create table t1 (a int, b numeric, c bigint);
    create index idx1 on t1(a);
    create index idx2 on t1(b, a);
    select * from t1 where a = 1 and b = 2;
```

I try to make it use idx2 instead of idx1.